ssh/build.json

```
@@ -1,13 +1,13 @@
 {
   "build_from": {
-    "aarch64": "homeassistant/aarch64-base:3.12",
-    "amd64": "homeassistant/amd64-base:3.12",
-    "armhf": "homeassistant/armhf-base:3.12",
-    "armv7": "homeassistant/armv7-base:3.12",
-    "i386": "homeassistant/i386-base:3.12"
+    "aarch64": "homeassistant/aarch64-base:3.13",
+    "amd64": "homeassistant/amd64-base:3.13",
+    "armhf": "homeassistant/armhf-base:3.13",
+    "armv7": "homeassistant/armv7-base:3.13",
+    "i386": "homeassistant/i386-base:3.13"
   },
   "args": {
-    "CLI_VERSION": "4.9.0",
+    "CLI_VERSION": "4.10.0",
     "LIBWEBSOCKETS_VERSION": "v3.2.2",
     "TTYD_VERSION": "1.6.0"
```

Tested this yesterday, already distributed with the community version (not other changes needed).

```suggestion
    "LIBWEBSOCKETS_VERSION": "v4.1.4",
    "TTYD_VERSION": "1.6.3"
```